src/applications/disability-benefits/2346/sass/form-2346.scss

```diff
@@ -7,6 +7,12 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
 @import "../../../../platform/forms/sass/m-form-confirmation";
 
+.personal-info-box {
+  margin-left: 0rem;
+  border-left: 8px solid #0071bb;
+  padding-left: 2rem;
+}
```

Comment:
It's better practice to add the design system class names to the element instead of here
- margin-left - `vads-u-margin-left--0` - ref: https://design.va.gov/utilities/margins#margin-left
- border size & color: this might be an exception since the defined styles don't exactly match what's available (ref: https://design.va.gov/utilities/border), but you will need to use the color variable `$color-primary` - ref: https://design.va.gov/utilities/color
- padding-left - `vads-u-padding-left--0` - ref: https://design.va.gov/utilities/padding#padding-left